compiler/ghc/default.nix

```diff
@@ -221,7 +222,7 @@ in let configured-src = stdenv.mkDerivation (rec {
 
         outputs = [ "out" ];
         phases = [ "unpackPhase" "patchPhase" ]
-              ++ stdenv.lib.optional (ghc-patches != []) "autoreconfPhase"
+              # ++ stdenv.lib.optional (ghc-patches != []) "autoreconfPhase"
```

Comment:
This seems dangerous. It will negate any patch that touches the autoconf files.

Reply:
This was commented out to get me around over some other issue I caused, and isn't actually necessary.